### ISSUE TITLE
Ἠραγόρης, Ἀγασίων, Ἀγασικράτης : Hi, Magdalena. No access to modify these names and delete their doublets

### ISSUE THREAD

**Issue by Hippodamie** [eXistSolutions/LGPN-ling#285]
Hi, Magdalena,

We have to modify these 3 names. But we cannot do anything to re-edit them. It does not work.
Could you, please, help ?
And there are doublets for them that we would like to delete.

Many thanks.
Best,
Sophie

**Comment by tuurma** [comment]
I will look into it and let you know.

**Comment by tuurma** [comment]
I've found 10 names with that problem
Agasiklēs
Agasiphōn
Agasimenēs
Agasisthenēs
Hagnodōros
Agasilēwos
Agassigitōn
Agasikratēs
Agasiōn
Hēragorēs

Trying to determine now how the dublets appear

**Comment by Hippodamie** [comment]
Hi, Magdalena !

Yes, inded, there are some which we weren’t able to modify since months.
I think doublets happened sometimes when the Edit window did not want to save anymore. So we tried to open another window to edit the same name from the beginning. But when we tried to delete the first try, it was never possible to delete it. So that 2 editions remain for the same name.

Thanks for your help.
Best wishes,

Sophie
